### PR TITLE
[Doc Improvement][Add details for Get app content stage sharing capabilities API][2487612]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/51db519caee5ca6ad781655346bde4a2d9fb9a7d/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 29 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8596d39f-250d-486d-9011-a0dbb7a0ff64) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/90d15e1681cc93c2b2e8fdef59bd0db37012b00b/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 92 | 29 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e7513431-5875-475a-b71d-4747bb7da620) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/eceb2219e6e9458336427639c74e37393f37a60a/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 91 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/681e624c-a113-49bc-91dc-5a2304e78287) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/acf3b19894817d1775891f71442bab00198544eb/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 91 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/909a9b9e-27e4-4314-8d35-2654c335bcec) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fdff88e0fd49e225d9727a200073fa34957fabf5/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 91 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b127d9e9-fe4e-4d89-852e-58ddb32f53e8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/294d1c92fe3e674455c3c936ec0a01f99d2817b5/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 91 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/56fb0bd5-5a29-494a-b588-3971a3b7b81c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/91fdc60b4db478d93454584f9c78592cb502e593/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 91 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bc1fff5b-c2b5-49cb-a1b8-2e2cf51aac3b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4f5d7a2bffda693d9a20a010eee00ad8f8082f9a/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 91 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5fc99b2b-268b-4306-a767-f63e66a3ad21) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/54dfe300ddd1bdaaa2664f6ce58eddd09a527a6b/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 91 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7c878922-5194-4680-ba0c-fb2141860d0f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a218603eb3258f495aac1aa572e64f38316b4913/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 91 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e6bc5ed2-7f0e-4f0f-a3ef-e1c4b44b8fd6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cc4ed3fcc3c25feed5b153d7d7950f111f5a209e/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 91 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/13a90f5f-1f6f-4c7c-97f7-dc9689864e58) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/01fd9b922aa9b1db7bc9d553486ba77238106ded/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 91 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e4b35ef2-c3cc-4651-b936-b5f62d3afb63) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/194714c69c2461341d6af6593cb32f9ef951f806/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 91 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/952caf4a-f70b-4286-8295-f87afa0647c9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/871ff5db281b3e619db1346c29cdfeed216b1e3f/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 91 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/aa8859ab-15a3-426b-92b6-2c09e328ce11) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2f67c05b8083d444b80d794cf8721c1661bda665/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 91 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fe39afc9-897f-4b45-88ff-dcf50679f4ff) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/apps-in-teams-meetings/API-references.md [hunk 1]
```diff
@@ -694,7 +694,13 @@ The following table provides the response codes:
 
 ## Get app content stage sharing capabilities API
 
-The `getAppContentStageSharingCapabilities` API enables you to fetch the app's capabilities for sharing to meeting stage.
+The `getAppContentStageSharingCapabilities` API enables you to fetch the app's capabilities for sharing to meeting stage. The `getAppContentStageSharingCapabilities` API  enables the app to render the share to stage button. If the app has sharing capabilities, then the button appears.
```
Comment (juna-gupta) [inline msteams-platform/apps-in-teams-meetings/API-references.md:697]: The share to stage button will not appear if a meeting participant doesn't have permission to share app content to the meeting stage.

Applications should call this API to enable/disable custom share to stage buttons for a meeting participant in the meeting side panel.

File: msteams-platform/apps-in-teams-meetings/API-references.md [hunk 2]
```diff
@@ -694,7 +694,13 @@ The following table provides the response codes:
...
+
+The app sharing capabilities depends on the tenant user type and participant roles in a meeting.
+
+* **User type**: In-tenant, guest, and external user type participants can share the app to stage and also see and interact with the app being shared on stage. Anonymous user can't see, share, or interact with the app that is being shared on the stage. For more information on user types, see [user types in a meeting.](~/apps-in-teams-meetings/meeting-app-extensibility.md#user-types-in-a-meeting)
+
+* **Participant roles**: Presenter and organizer meeting role participants can share the app to stage. Attendee won't have the share to stage button enabled and ability to share the app to stage. For more information on participant roles, see [participant roles in a meeting.](~/apps-in-teams-meetings/meeting-app-extensibility.md#participant-roles-in-a-meeting)
 
 ### Query parameter
```
Comment (juna-gupta) [inline msteams-platform/apps-in-teams-meetings/API-references.md:706]: Update the comment in the JS example:

"Indicates if the meeting participant has permission to share content to the meeting stage."

**Review by juna-gupta — CHANGES_REQUESTED** [review]
Please address the comments.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/API-references.md](https://github.com/MicrosoftDocs/msteams-docs/blob/61be35309260c48b220712cafdb78954340057e1/msteams-platform%2Fapps-in-teams-meetings%2FAPI-references.md) | 91 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/36e5a55b-c4b9-4f35-ac82-bd734629b169) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md](https://github.com/MicrosoftDocs/msteams-docs/blob/720d72323e35a41ec1489701c8518fc28aaa921a/msteams-platform%2Fapps-in-teams-meetings%2Fbuild-apps-for-teams-meeting-stage.md) | 90 | 19 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d0cb33be-8390-417d-ad7b-c024651463b3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4fa4aff90c5f33e8d65557a527fbdda85e7b3112/msteams-platform%2Fapps-in-teams-meetings%2Fbuild-apps-for-teams-meeting-stage.md) | 90 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4e430988-2cd0-4826-9c7b-458c13cac8c8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md](https://github.com/MicrosoftDocs/msteams-docs/blob/47ba92652656acd96b39c968000792820de9691b/msteams-platform%2Fapps-in-teams-meetings%2Fbuild-apps-for-teams-meeting-stage.md) | 90 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9598af6e-fdde-4f72-b279-9df0090eb441) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md [hunk 3]
```diff
@@ -148,7 +148,13 @@ The following table provides the response codes:
 
 ## Get app content stage sharing capabilities API
 
-The `getAppContentStageSharingCapabilities` API enables you to fetch the app's capabilities for sharing to meeting stage.
+The `getAppContentStageSharingCapabilities` API enables you to fetch the app's capabilities for sharing the app content to meeting stage. Apps need to call the `getAppContentStageSharingCapabilities` API to either enable or disable the custom share to stage button for a meeting participant in the meeting side panel. The share to stage button will not appear if a meeting participant doesn't have permission to share the app content to meeting stage.
```
Comment (juna-gupta) [inline msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md:151]: share to stage button should be disabled/hidden if a meeting participant doesn't have permission to share the app content to the meeting stage.

**Review by juna-gupta — DISMISSED** [review]
Minor fix in the comments

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md](https://github.com/MicrosoftDocs/msteams-docs/blob/51d22441bc39295ff8e5147d6d95b26bf6f93613/msteams-platform%2Fapps-in-teams-meetings%2Fbuild-apps-for-teams-meeting-stage.md) | 90 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/94ceb19d-3502-4ad3-bb53-e3d0d7facf5e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b0d694c64d8738b446f185309a72beaef2a34ff6/msteams-platform%2Fapps-in-teams-meetings%2Fbuild-apps-for-teams-meeting-stage.md) | 90 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/de5cc8c2-f01b-4d48-b7f1-97108a7e23ad) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b0719d3b5b88b589f4ccc7c79cc1436ec1a1cb2e/msteams-platform%2Fapps-in-teams-meetings%2Fbuild-apps-for-teams-meeting-stage.md) | 91 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4dbeb056-60ea-48f1-b709-bda38b183971) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2137934a0483ef6a11690acc22348cd6549a6ff9/msteams-platform%2Fapps-in-teams-meetings%2Fbuild-apps-for-teams-meeting-stage.md) | 91 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bd4826e7-8ce1-46cc-ae28-eaf3ce56fa36) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**